### PR TITLE
chore: Revert two typo fixes in DEPENDENCIES.md since it is auto-generated.

### PR DESCRIPTION
**Pull request by Sajjon**
As requested by @linabutler, I've reverted the changes in the auto-generated `DEPENDENCIES.md`, introduced in #6251.

I apologies @linabutler and @mhammond I misunderstood you in 6251, I thought since the PR was merged that there was no action needed....

### CODE REVIEW

**Review by mhammond — APPROVED** [review]
Thanks, and no worries - the real problem was me approving it when I should have spotted the issue Lina did :)

**Comment by linabutler** [comment]
No worries at all, @Sajjon; no apology needed—thank _you_ for fixing up all those typos in the original (and for your contributions to UniFFI!) 👏🏻 

I should've clarified that it would've been OK to leave your fixes to `DEPENDENCIES.md` after merging—I think they would've been automatically reverted on the next run of `python ./tools/dependency_summary.py > DEPENDENCIES.md`, anyway.

**Pull request merged**
